bleak/backends/bumble/utils.py

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2024 Victor Chavez
+def uuid_bytes_to_str(byte_data: bytes) -> str:
```

Comment:
Why not use the `uuid` module from the standard library for this?

Reply:
I should have made this more clear. The function should convert data bytes from the custom UUID class from bumble (bumble.core.UUID). This is not a standard uuid and requires a conversion.

Example

- **Shortened 16 bit UUID**: 0x1800
- **bumble UUID string representation**: 'UUID-16:1800 (Generic Access)'
- **bumble UUID bytes representation:** b'\x00\x18'